package.json

```diff
@@ -6,10 +6,10 @@
         "build": "nx build",
         "test": "nx test",
         "lint": "nx lint",
-        "start:all": "nx run-many --target=serve --all",
-        "build:all": "nx run-many --target=build --all",
-        "lint:all": "nx run-many --target=lint --all",
-        "test:all": "nx run-many --target=test --all --coverage",
+        "start:all": "nx run-many --target=serve --projects=api,client,dashboard,contracts --parallel=4",
```

Comment:
Note: `--all` is deprecated as per their docs. By default only 3 projects can be run in parallel, so added the `--parallel=4` arg.

Reply:
We can also use the `--projects=*` syntax here.